beginners/easy/2.positive-negative/802.w3samiulazim.positiveNegative.c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+int main(){
+
+    float a;
+    scanf("%f", &a);
+
+    if(a > 0){
+        printf("This is a positive number");
```

Comment:
must add new line end of the statement

Reply:
mistake, will take care of it next time.